docs/G-Codes.md

```diff
@@ -1202,6 +1202,30 @@ print.
 #### SDCARD_RESET_FILE
 `SDCARD_RESET_FILE`: Unload file and clear SD state.
 
+### [z_thermal_adjust]
+
+The following commands are available when the
+[z_thermal_adjust config section](Config_Reference.md#z_thermal_adjust)
+is enabled.
+
+#### QUERY_Z_THERMAL_ADJUST
+`QUERY_FRAME_COMP`: reports current state and key parameters of the Z Thermal
+Adjust module.
+
+#### SET_Z_THERMAL_ADJUST
+`SET_Z_THERMAL_ADJUST [ENABLE=<0:1>] [COEFF=<value>]`: Enable or disable the
+Z thermal adjustment with `ENABLE`. Disabling does not remove any adjustment
+already applied, but will freeze the current adjustment value - this prevents
+potentially unsafe downward Z movement. Re-enabling can potentially cause upward
+tool movement as the adjustment is updated and applied. `COEFF` allows run-time
+tuning of the adjustment temperature coefficient (i.e. the `TEMP_COEFF` config
+parameter). `COEFF` values are not saved to the config.
+
+#### SET_Z_THERMAL_ADJUST_REF
+`SET_Z_THERMAL_ADJUST_REF [TEMPERATURE=<value>]`: Set the thermal adjustment
+reference temperature to the current reading, or to the value supplied with
+`TEMPERATURE`.
```

Comment:
I think it would be preferable to use a single command for all three of the above - SET_Z_THERMAL_ADJUST with no parameters just reports the current state, SET_Z_THERMAL_ADJUST with a REFERENCE_TEMPERATURE=123.4 sets the reference temperature, SET_Z_THERMAL_ADJUST with COEFF=567.8 sets the coefficient, etc.